mlrun/feature_store/retrieval/dask_merger.py

```diff
@@ -12,6 +12,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
+import re
+
 import dask.dataframe as dd
 import pandas as pd
 from dask.dataframe.multi import merge, merge_asof
```

Comment:
```suggestion
import dask.dataframe.multi as dask_df_multi 
```

Then use it like `dask_df_multi.merge` or `dask_df_multi.merge_asof`